javascripts/govuk/analytics/example-init/example-init.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  "use strict";
+
+  // Load Google Analytics libraries
+  GOVUK.Tracker.load();
+
+  // Use document.domain in dev, preview and staging so that tracking works
+  // Otherwise explicitly set the domain as www.gov.uk (and not gov.uk).
+  var cookieDomain = (document.domain === 'www.gov.uk') ? '.www.gov.uk' : document.domain;
+
+  // Configure profiles, setup custom vars, track initial pageview
+  // and make interface public for virtual pageviews and events
+  GOVUK.analytics = new GOVUK.Tracker({
+    universalId: 'UA-XXXXXXXX-X',
+    classicId: 'UA-XXXXXXXX-X',
+    cookieDomain: cookieDomain
+  });
+
+  // Set custom dimensions before tracking pageviews
+  // GOVUK.analytics.setDimension(…)
```

Comment:
Is it worth adding to the comment, the URL to the doc section about using setDimension? Just so they know which params to pass to it.